check_elasticsearch_cluster.pl

```diff
@@ -148,6 +148,16 @@ ($$)
   $np->add_message($code, $_[1]);
 }
 
+sub get_threshold_value {
+  my ($thresh, $key) = @_;
```

Comment:
Needs `$k` as argument too.

Use `$threshold` instead for clarity. It's not too verbose and explains more what it is.

Also, `$key` is actually a value (confusing, I know) so maybe it should be:
```perl
my ($threshold, $key, $value) = @_;
```